backend/inspirehep/records/api/mixins.py

```diff
@@ -373,6 +374,16 @@ def create_conferences_relations(self, document_type):
                         ),
                     )
                 )
+        if len(conference_literature_relations_waiting_for_commit) > 0:
+            db.session.bulk_save_objects(
+                conference_literature_relations_waiting_for_commit
+            )
+        LOGGER.info(
+            "Conferecnce documents references set",
```

Comment:
should this be inside the if? Also, maybe something like ```Added conference-literature relations```

Reply:
:+1: